eng/deploy.yaml

```diff
@@ -20,9 +21,20 @@ variables:
 stages:
 - stage: deploy
   displayName: Deploy
-  dependsOn: build
+  dependsOn:
+  - build
   jobs:
+  - deployment: approval
+    displayName: deployment approval (conditional)
+    environment: ${{ parameters.DeploymentEnvironment }}
```

Comment:
This means we will have approvals for staging deployments too, is that the intention here?

Reply:
My understanding is the "requires approval" is a thing in Azure Devops. An environment can be set to require them or not.  We should have the Staging environment to not require that.

Reply:
I added staging and made sure there are no checks for it. :-)

Reply:
Staging environment with no checks sounds fine.